add new personalities to code

ramcq opened the pull request:
Not sure how to add these to the translation lists - apparently we do want to still show Spanish (Guatemala) so I've added that back too.

https://phabricator.endlessm.com/T12946

wjt commented on src/endless/EndlessUsbToolDlg.cpp:
```diff
@@ -179,23 +179,27 @@ DWORD usbDevicesCount;
 static const wchar_t *globalAvailablePersonalities[] =
 {
+    PERSONALITY_ARABIC,
     PERSONALITY_BASE,
```
Unfortunately the order of this array is significant: [exhibit A](https://github.com/endlessm/rufus/blob/9deefca1dc8c9a1e8ac8c156f1ce7e65064eba04/src/endless/EndlessUsbToolDlg.cpp#L472), [exhibit B](https://github.com/endlessm/rufus/blob/9deefca1dc8c9a1e8ac8c156f1ce7e65064eba04/src/endless/res/endless.loc#L232) so you can't rearrange these in alphabetical order without also fixing all the translations.

wjt commented on src/endless/EndlessUsbToolDlg.cpp:
```diff
@@ -179,23 +179,27 @@ DWORD usbDevicesCount;
 
 static const wchar_t *globalAvailablePersonalities[] =
 {
+    PERSONALITY_ARABIC,
```
https://github.com/endlessm/rufus/blob/9deefca1dc8c9a1e8ac8c156f1ce7e65064eba04/src/endless/res/endless.loc#L232

wjt replied:
Apparently I can't drive GitHub, but my point is that the order of this array matters, so we can't sort it into alphabetical order without rearranging all the translations.

https://github.com/endlessm/rufus/blob/9deefca1dc8c9a1e8ac8c156f1ce7e65064eba04/src/endless/res/endless.loc#L232

wjt replied:
https://github.com/endlessm/rufus/blob/9deefca1dc8c9a1e8ac8c156f1ce7e65064eba04/src/endless/EndlessUsbToolDlg.cpp#L472

wjt committed:
Fix spelling of "portughese"

wjt committed:
Support bn and es_GT images

We don't have a separate translation for es_GT (here or in Endless OS) so
continue to default to global Spanish for that locale.

wjt commented:
This branch is on top of #8, so shouldn't really be reviewed/merged till that one is merged.
